tests/data/source.py

```diff
@@ -0,0 +1,18 @@
+class Square:
```

Comment:
yeah.  I hesitated adding a full file for testing, but I think this is cleaner.  Nice!